Sources/zcash-swift-payment-uri/Model.swift

```diff
@@ -141,7 +150,23 @@ public struct MemoBytes {
         
         self.data = memoStringData
     }
-    
+
+    public init(base64URL: String) throws {
+        var base64 = base64URL.replacingOccurrences(of: "_", with: "/")
+            .replacingOccurrences(of: "-", with: "+")
```

Comment:
What are these replacements for?

Reply:
So the `Data(base64Encoded: base64)` does not fail
References: 
https://stackoverflow.com/questions/43499651/decode-base64url-to-base64-swift
https://en.wikipedia.org/wiki/Base64#Variants_summary_table
https://en.wikipedia.org/wiki/Base64#URL_applications